chromadb/test/property/test_collections.py

```diff
@@ -45,10 +46,10 @@ def create_coll(self, coll):
         if coll["name"] in self.existing:
             with pytest.raises(Exception):
                 c = self.api.create_collection(**coll)
-            c = self.api.get_collection(name=coll["name"])
-        else:
-            c = self.api.create_collection(**coll)
-            self.existing.add(coll["name"])
+            return multiple()
```

Comment:
What does this do?

Reply:
Returns from the rule without adding anything to the bundle.